recipe/meta.yaml

```diff
@@ -66,9 +64,9 @@ test:
     - {{ compiler('cuda') }}
     - {{ stdlib('c') }}
     # need the libcuda stub for import test
-    - cuda-driver-dev                          # [linux and (cuda_compiler_version or "").startswith("12")]
+    - cuda-driver-dev  # [linux and ((cuda_compiler_version or "").startswith("12") or (cuda_compiler_version or "").startswith("13"))]
   files:
-    - test_load_elf.c        # [linux]
+    - test_load_elf.c  # [linux]
```

Comment:
We should probably update this test at some point.

Reply:
Yes, we can use the tools that are already build dependencies.